packages/rescript-relay/__tests__/__generated__/TestConnections_user_graphql.res

```diff
@@ -51,10 +51,11 @@ let connectionKey = "TestConnections_user_friendsConnection"
 )
 
 @live
-let makeConnectionId = (connectionParentDataId: RescriptRelay.dataId, ~onlineStatuses: array<RelaySchemaAssets_graphql.enum_OnlineStatus>=[Idle, Offline], ~beforeDate: TestsUtils.Datetime.t) => {
+let makeConnectionId = (connectionParentDataId: RescriptRelay.dataId, ~onlineStatuses: array<RelaySchemaAssets_graphql.enum_OnlineStatus>=[Idle, Offline], ~beforeDate: TestsUtils.Datetime.t, ~orderBy: array<RelaySchemaAssets_graphql.input_UserOrder>=Obj.magic({"direction": ASC, "field": FIRST_NAME})) => {
```

Comment:
It'll fail with the following error on the line

<img width="884" alt="image" src="https://github.com/zth/rescript-relay/assets/49292466/6235efea-b365-4a84-b823-60dc09e38a19">